### DISCUSSION TITLE
Can pages be hidden from navigation?

### DISCUSSION THREAD

**Discussion by timlonggr** [squidfunk/mkdocs-material#5856] [Q&A]
I have a lot of individual pages of documentation, enough that listing all of them makes the side navigation pretty difficult to use. But I don't want to delete those pages and want them available in the search. Is there a way to hide pages from the side nav, or to create a completely custom side bar so I can just add the items I want?

**Comment by squidfunk** [comment]
Without further context, it sounds to me that you're not defining [`nav`](https://www.mkdocs.org/user-guide/configuration/#nav) in `mkdocs.yml`. Without that, MkDocs will build the navigation by itself and add all pages. You could either define `nav`, or use a plugin like [`awesome-pages`](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin) to configure navigation, or use the new [`not_in_nav`](https://www.mkdocs.org/user-guide/configuration/#not_in_nav) setting (MkDocs 1.5+) once https://github.com/mkdocs/mkdocs/issues/3336 is resolved.

You can upvote or comment on the issue if you like.
